code-challenges.test.js

```diff
@@ -49,4 +116,41 @@ const hand3 = [5, 5, 5, 5, 4]
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
+// RED
+describe("fullHouse", () => {
+    it("takes in an array of numbers and determines if it is a full house (one pair, one three of a kind)", () => {
+        expect(fullHouse(hand1)).toEqual(true)
+        expect(fullHouse(hand2)).toEqual(false)
+        expect(fullHouse(hand3)).toEqual(false)
+        expect(fullHouse(hand4)).toEqual(true)
+    })
+})
+
+
 // b) Create the function that makes the test pass.
+
+// GREEN
+// create a function that takes in an array of numbers and determines whether those numbers make a full house (one pair, one three of a kind)
+// input: an array of numbers
+// output: boolean value
+// first, sort the array from least to greatest using .sort()
+// then, using square bracket notation on the sorted array (orderedArray[0]), determine whether the values match on index zero and one, AND that they match on two, three, and four
+
+const fullHouse = (array) => {
+    let orderedArray = array.sort((a, b) => a - b)
+    if (orderedArray[0] === orderedArray[1] && orderedArray[2] === orderedArray[3] && orderedArray[3] === orderedArray[4]) {
+        return true
+    } else {
+        return false
+    }
+}
+console.log(fullHouse(hand1))
+// output: true
+console.log(fullHouse(hand2))
+// output: false
+console.log(fullHouse(hand3))
+// output: false
+console.log(fullHouse(hand4))
+// output: true
+
+// Note: I did some research on this one before hand, but ultimately the methods I found online seemed far more involved than what I was looking to achieve. Doing it my way, I can encapsulate everything in one function without having to jump through hoops or go back and forth to determine whether the array values match. Sorting them from greatest to least would also work, you would simply have to flip your if statement to have indexes 0, 1 and 2 matching, as well as 3 and 4.
```

Comment:
No, this is awesome! I was going to leave a comment that I really liked your approach. Great job.